### ISSUE TITLE
[calcite-switch] Not operable by keyboard in Windows + Chrome + NVDA

### ISSUE THREAD

**Issue by fdeters** [Esri/calcite-design-system#13952]
**monday.com sync:** #11318398289

### Check existing issues

- [x] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Which Component

Switch

### Actual Behavior

The switch control does not respond to input from Enter or Space while NVDA is active (on Chrome on Windows). This behavior does not occur on Firefox on Windows.

### Expected Behavior

I expect the switch to respond to input while a screen reader is active.

### Reproduction Sample

https://developers.arcgis.com/calcite-design-system/components/switch/

### Reproduction Steps

1. Open the reproduction sample in Chrome in Windows (not sure if the issue will happen on other OSes).
2. Start NVDA.
3. Tab to the calcite-switch.
4. Press Space and Enter.
5. See that the switch has not changed state.
6. Click on the switch.
7. See that it changes state.

### Reproduction Version

5.0.2

### Relevant Info

Windows 11
Chrome 145.0.7632.109
NVDA 2025.3.1.53219

### Priority impact

impact - p2 - want for an upcoming milestone

### Impact

Users who rely on NVDA + Chrome to navigate our site will be unable to operate switch-based controls.

### Calcite package

- [x] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/calcite-ui-icons
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

N/A

**Comment by geospatialem** [comment]
Related to the functionality from https://github.com/Esri/calcite-design-system/issues/11996. This is blocked by some additional research needed to determine next steps, since it is likely due to support with web components across browsers and assistive technologies. More information available via [this comment](https://github.com/Esri/calcite-design-system/issues/11996#issuecomment-2977506468).

In our initial findings we have found both `checkbox` and `switch` will work with either of the following combinations (although not ideal to support more audiences):
- `Shift` + `Enter` keys
- When the screen reader is in forms mode

**Comment by janett-baresel** [comment]
This got reported in Scene Viewer.

**Comment by bethanyscott** [comment]
Just noting that this was also [reported as part of the Data Pipelines](https://devtopia.esri.com/WebGIS/arcgis-web-analysis/issues/11121) accessibility assessment.
